pygenometracks/getAllDefaultsAndPossible.py

```diff
@@ -66,16 +83,20 @@ def main():
             all_possible_parameters[p][track_type] = ["true", "false"]
 
     # For the default they are summarized in a matrix
-    mat = np.empty((len(all_default_parameters) + 2, len(all_tracks) + 1),
+    mat = np.empty((len(all_default_parameters) + 2, len(all_tracks_with_default) + 1),
                    dtype='U25')
     mat[0, 0] = 'parameter'
     mat[1, 0] = '--'
     for j, track_type in enumerate(all_tracks_with_default, start=1):
-        mat[0, j] = track_type
+        mat[0, j] = f":doc:`tracks/{track_type}`"
         mat[1, j] = '-'
         for i, p in enumerate(all_default_parameters):
             if j == 1:
-                mat[i + 2, 0] = p
+                if p in putStarsAfter:
```

Comment:
why?

Reply:
I put a star after second_file and operation to encourage people to preprocess their data...

Reply:
This was proposed by Bjorn at the end of https://github.com/deeptools/pyGenomeTracks/pull/165

Reply:
I see